lib/src/components/NxPageHeader/__tests__/NxPageHeader.test.tsx

```diff
@@ -5,84 +5,117 @@
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
 import React from 'react';
-import { shallow } from 'enzyme';
-import 'jest-enzyme';
-
-import * as enzymeUtils from '../../../__testutils__/enzymeUtils';
 import NxPageHeader from '../NxPageHeader';
-import AbstractNxPageHeader from '../../AbstractNxPageHeader/AbstractNxPageHeader';
+
+import { screen } from '@testing-library/react';
+import { rtlRenderElement } from '../../../__testutils__/rtlUtils';
 
 describe('NxPageHeader', function() {
-  const getShallowComponent = enzymeUtils.getShallowComponent(NxPageHeader, {});
 
-  it('renders an AbstractNxPageHeader', function() {
-    expect(getShallowComponent()).toMatchSelector(AbstractNxPageHeader);
+  const renderEl = rtlRenderElement(NxPageHeader, {});
+
+ it('renders a top-level header element', function() {
+    renderEl();
+    const header = screen.getByRole('banner');
+    expect(header).toBeInTheDocument();
   });
 
-  it('passes its links, homeLink, and children props to the AbstractNxPageHeader', function() {
-    const props = {
-          links: [{ name: 'foo', href: '#bar' }],
-          homeLink: '#home',
-          children: <button>Click Here For a Free iPhone</button>
-        },
-        component = getShallowComponent(props);
+  it('renders the default logo within the header', function() {
+    renderEl();
+    const logo = screen.getByRole('img');
+    expect(logo).toBeInTheDocument();
+  });
+
+  it('renders a custom logo instead of the default one when given logo prop', function() {
+    renderEl({ logo: { path: 'foo', alt: 'bar' } });
+    const logo = screen.getByRole('img');
+
+    expect(logo).toHaveAttribute('src', 'foo');
+    expect(logo).toHaveAttribute('alt', 'bar');
+    // ensures there is only one logo
+    expect(screen.getAllByRole('img').length).toEqual(1);
```

Comment:
`getByRole`, which you're using above, already verifies that there's only one `img`.  RTL `getBy…` queries throw an exception if they find more than one matching element.

Reply:
> `getByRole`, which you're using above, already verifies that there's only one `img`. RTL `getBy…` queries throw an exception if they find more than one matching element.

I removed the test here: [e13c730](https://github.com/sonatype/sonatype-react-shared-components/pull/935/commits/e13c730c802bc0e5b9c89013095c51200fedc0b2)